Test/MemoryTestList.txt

```diff
@@ -0,0 +1,3 @@
+[../Falcor.sln {Released3d12 ..\Bin\x64\Release\}
+ShaderBuffers {-test -memchecktimes 240-3840 -ssframes 50 -shutdowntime 4000} {released3d12}
```

Comment:
I don't care about ShaderBuffer memory check. That should be the feature demo with the sun-temple